pkg/upgrade/upgrader/cstor_volume_upgrade.go

```diff
@@ -321,6 +314,15 @@ func getCV(pvLabel, namespace string) (*apis.CStorVolume, error) {
 	if len(cvList.Items) == 0 {
```

Comment:
We know there will be one to one map between PV and cStorVolume. Can we have check 
if len(cvList.Items) != 1 { return error }